### ISSUE TITLE
create a github action (or something else) to auto-assign date labels to new issues

### ISSUE THREAD

**Issue by lpmi-13** [remotehack/remotehack.github.io#30]
currently, I'm manually adding monthly labels to the issues, depending on which hackday cycle they got raised in (eg, for everything raised after the April hackday, but before the May hackday, I'm going to give it a label of May 2020.

It would be much nicer if we could get the system to automatically assign a label based on when the issue is raised.

possible links to investigate existing approaches to this:

https://github.com/nerrad/triage-new-issues

https://github.com/apps/enhanced-triage-bot

https://github.com/probot/probot

**Comment by dstockstad** [comment]
I might take a stab at this one.

**Comment by dstockstad** [comment]
Testing Github integration

**Comment by lpmi-13** [comment]
I guess one of the possible edge cases is that if somebody raises an issue that ISN'T a specific hack idea, we might not necessarily want to assign a label to that...then again, if we just auto assign the labels based on the date, people could always add other labels if they wanted.

**Comment by dstockstad** [comment]
Hmm, that's a good point. We could either create a separate repo for hackideas or possibly have people add a label to the issue called hackidea and we'd only operate on those issues. Maybe, I don't actually know what I'm doing :) Never used Github actions.

**Comment by lpmi-13** [comment]
yeah, right, I had kinda been thinking we could have a special label added for something that needs to be labelled, but then it's like, why wouldn't we just ask the people to add the label...maybe the easiest and hackiest thing to do is just to auto-label it IF it doesn't have a label that's been manually added...or the manually added label isn't a date label?

**Comment by lpmi-13** [comment]
Claude basically one shotted this

![Image](https://github.com/user-attachments/assets/6a2ada1a-37b1-45c7-8a4f-03fd9a5593b6)
